membership/src/main/scala/zio/membership/hyparview/InitialProtocol.scala

```diff
@@ -0,0 +1,76 @@
+package zio.membership.hyparview
+
+import upickle.default._
+import zio.membership.ByteCodec
+
+sealed abstract class InitialProtocol[T]
```

Comment:
Style-wise, name the param `A` or use more verbose name if needed.

Reply:
Will do. I used T for the node type for now, but I'll do one pass cleaning all of them up